### PR TITLE
Optimization of metrics usage in routes

### PR DESCRIPTION
**Pull request by gpotter2**
Small optimization in routes-ordering.

Better consistency in `inet/inet6.route()` with:
- `dst` argument (it was previously  `dst` or `dest`)
- `verbose` argument. It will allow people to hide the IPv6 no route found message, as on IPv4
- passing None returns the default route

### CODE REVIEW

**Comment by guedou** [comment]
Is it cover by unit tests? I did not manage to find something where we check the behaviour of this part of the code. If not, could you add one and compare the current and post-PR behavior?

Thanks.

**Comment by gpotter2** [comment]
It’s not directly covered (indirectly by networking tests).

I’ll add tests

File: scapy/route.py [hunk 1]
```diff
@@ -126,54 +125,55 @@ def ifadd(self, iff, addr):
         the_net = the_rawaddr & the_msk
         self.routes.append((the_net, the_msk, '0.0.0.0', iff, the_addr, 1))
 
-    def route(self, dest, verbose=None):
-        if dest is None:
-            dest = "0.0.0.0"
-        elif isinstance(dest, bytes):
+    def route(self, dst, verbose=conf.verb):
```
Comment (p-l-) [inline scapy/route.py]: I think it is helpful that `route(None)` gives the default route.

Comment (gpotter2) [reply scapy/route.py]: Could be, but that’s inconsistent with route6 :/ should I add it for both ?

Comment (p-l-) [reply scapy/route.py]: That's my opinion yes

**Comment by p-l-** [comment]
Thanks @gpotter2!

**Pull request merged**